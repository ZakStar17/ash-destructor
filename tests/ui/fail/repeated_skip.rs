use ash_destructor::DeviceDestroyable;

struct ImplDeviceDestroyable {}

impl DeviceDestroyable for ImplDeviceDestroyable {
    unsafe fn destroy_self(&self, _device: &ash::Device) {}
}

#[derive(DeviceDestroyable)]
struct Named {
    a: ImplDeviceDestroyable,
    #[skip]
    #[skip]
    b: String,
}

#[derive(DeviceDestroyable)]
struct Unnamed(
    ImplDeviceDestroyable,
    #[skip]
    #[skip]
    String,
);

fn main() {}
