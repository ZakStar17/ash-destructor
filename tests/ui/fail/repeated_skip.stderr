error: Multiple #[skip] attributes on a single field
  --> tests/ui/fail/repeated_skip.rs:12:5
   |
12 | /     #[skip]
13 | |     #[skip]
14 | |     b: String,
   | |_____________^

error: Multiple #[skip] attributes on a single field
  --> tests/ui/fail/repeated_skip.rs:20:5
   |
20 | /     #[skip]
21 | |     #[skip]
22 | |     String,
   | |__________^
